karabo_data/reader.py

```diff
@@ -30,7 +30,6 @@
 from .read_machinery import (
     DETECTOR_SOURCE_RE,
     DataChunk,
-    FilenameInfo,
```

Comment:
This one was deliberately left despite being unused, in case other code outside karabo_data is using it (`from karabo_data.reader import FilenameInfo`)

Reply:
aha, I'll put it back